rlclientlib/logger/http_transport_client.h

```diff
@@ -66,7 +66,13 @@ class http_transport_client : public i_sender
     int join();
 
   private:
-    pplx::task<web::http::status_code> send_request(size_t try_count);
+    // repeat request until success or _max_retries attempted
+    // returns the http::status_code of the final attempt.
+    pplx::task<web::http::status_code> send_request();
```

Comment:
is this comment correct?

Reply:
Yes.  send_request() returns the status code field of the response object, send_request_with_retries() returns the full response object.